lib/Test/Perl/Critic.pm

```diff
@@ -23,6 +23,13 @@ my $TEST = Test::Builder->new;
 my $DIAG_INDENT = q{  };
 my %CRITIC_ARGS = ();
 
+my $CRITIC_OBJ = undef;
+my $GET_CRITIC = sub {
```

Comment:
don't use 'get' - my $BUILD_CRITIC

implementation:
```perl
    return $CRITIC_OBJ //= Perl::Critic->new( @_ )
```

Reply:
ok, my fault, apparently it should work also for perl < 5.10
still, assign to %args is useless

Reply:
@branislav-zahradnik-gdc Ok, I'll change that.